ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -19,7 +19,7 @@ def absolute(x):
     return ivy.abs(x)
 
 
-abs = absolute
+#abs = absolute
```

Comment:
why did you comment this out ?

Reply:
Hi sir, it appeared as part of the errors.